### PR TITLE
Platform 360 - Environmental Health Form Config

### PR DESCRIPTION
**Pull request by jamesgiegerich94**
### Description of proposed changes
_What does this pull request accomplish? Why should it be integrated?_

### Checklist
_Put an `x` in the boxes that apply._
- [ ] My lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my changes work (if applicable)
- [ ] I have added or updated necessary documentation (if appropriate)

### Additional comments
_Is there anything else you want us to know?_

### CODE REVIEW

**Comment by hopetambala** [comment]
@jamccomb92 this looks good to me! approve if it looks good to you

File: domains/DataCollection/Forms/SupplementaryForm/configs/envhealth.config.js [hunk 1]
```diff
@@ -122,18 +150,28 @@ const configArray = {
...
       },
       {
-        label: 'global.no',
-        value: 'No'
+        label: 'environmentalHealth.bathroomAccess.Latrine',
+        value: 'Latrine'
+      },
+      {
+        label: 'environmentalHealth.bathroomAccess.None',
+        value: 'N'
```
Comment (jamccomb92) [inline domains/DataCollection/Forms/SupplementaryForm/configs/envhealth.config.js:168]: Let's change this value to None

File: domains/DataCollection/Forms/SupplementaryForm/configs/envhealth.config.js [hunk 2]
```diff
@@ -156,9 +194,40 @@ const configArray = {
...
   {
-    label: 'environmentalHealth.clinicAccess',
+    label: 'environmentalHealth.clinicAccess.label',
     formikKey: 'clinicAccess',
     value: '',
+    fieldType: 'selectMulti',
+    options: [
+      {
+        label: 'environmentalHealth.clinicAccess.Clinic',
+        value: 'Clinic/Medical_Center'
```
Comment (jamccomb92) [inline domains/DataCollection/Forms/SupplementaryForm/configs/envhealth.config.js:204]: I'd prefer that the key does not include '/'

File: domains/DataCollection/Forms/SupplementaryForm/configs/envhealth.config.js [hunk 3]
```diff
@@ -208,36 +274,57 @@ const configArray = {
     options: [
       {
         label: 'global.yes',
-        value: 'Yes'
+        value: 'Y'
```
Comment (jamccomb92) [inline domains/DataCollection/Forms/SupplementaryForm/configs/envhealth.config.js:277]: Don't think we should be changing the value of question. All others collected will not match those previously collected. Leave as Yes

File: domains/DataCollection/Forms/SupplementaryForm/configs/envhealth.config.js [hunk 4]
```diff
@@ -208,36 +274,57 @@ const configArray = {
...
       },
       {
         label: 'global.no',
-        value: 'No'
+        value: 'N'
```
Comment (jamccomb92) [inline domains/DataCollection/Forms/SupplementaryForm/configs/envhealth.config.js:281]: Don't think we should be changing the value of question. All others collected will not match those previously collected. Leave as No

File: domains/DataCollection/Forms/SupplementaryForm/configs/envhealth.config.js [hunk 5]
```diff
@@ -208,36 +274,57 @@ const configArray = {
...
       }
     ],
     validation: false
   },
   {
     label: 'environmentalHealth.floorCondition.label',
-    formikKey: 'conditionoFloorinyourhouse',
+    formikKey: 'floorCondition',
```
Comment (jamccomb92) [inline domains/DataCollection/Forms/SupplementaryForm/configs/envhealth.config.js:288]: why are we changing the formikKey? all other values collected will be in separate field

File: domains/DataCollection/Forms/SupplementaryForm/configs/envhealth.config.js [hunk 6]
```diff
@@ -394,10 +485,30 @@ const configArray = {
...
-    fieldType: 'numberInput',
+    fieldType: 'selectMulti',
+    options: [
+      {
+        label: 'environmentalHealth.numberChildrenUnder5.under_5',
+        value: 'under_5'
+      },
+      {
+        label: 'environmentalHealth.numberChildrenUnder5.6_12',
+        value: '6-12'
```
Comment (jamccomb92) [inline domains/DataCollection/Forms/SupplementaryForm/configs/envhealth.config.js:499]: prefer consistent value 6_12

File: domains/DataCollection/Forms/SupplementaryForm/configs/envhealth.config.js [hunk 7]
```diff
@@ -394,10 +485,30 @@ const configArray = {
...
+      },
+      {
+        label: 'environmentalHealth.numberChildrenUnder5.13_18',
+        value: '13-18'
```
Comment (jamccomb92) [inline domains/DataCollection/Forms/SupplementaryForm/configs/envhealth.config.js:503]: 13_18

File: domains/DataCollection/Forms/SupplementaryForm/configs/envhealth.config.js [hunk 8]
```diff
@@ -156,9 +194,40 @@ const configArray = {
...
+      {
+        label: 'environmentalHealth.clinicAccess.Polyclinic',
+        value: 'Polyclinic'
+      },
+      {
+        label: 'environmentalHealth.clinicAccess.Hospital',
+        value: 'Hospital'
+      },
+      {
+        label: 'environmentalHealth.clinicAccess.None',
```
Comment (jamccomb92) [inline domains/DataCollection/Forms/SupplementaryForm/configs/envhealth.config.js:215]: change to None

**Review by jamccomb92 — CHANGES_REQUESTED** [review]
This all looks great! I would make the changes I commented on. Also in terms of all keys, I think we should stick to a consistent naming convention for values whether that be camelCase or underscores. I'd also avoid adding any special characters like '/' or '-'. It's a bit nit picky but I'd prefer consistent naming conventions where we can control, so I would ensure that all values match, maybe that wasnt the case before and if it isnt only make the requested changes. If all others are consistent camelCase or under_score lets stick to it throughout the form.

**Review by jamccomb92 — APPROVED** [review]
Everything looks great! thanks @jamesgiegerich94

**Review by hopetambala — APPROVED** [review]

**Pull request merged**